spec/models/events/review_comment_spec.rb

```diff
@@ -73,27 +84,33 @@
     end
 
     describe '#created' do
+      before { subject.payload['action'] = 'created' }
+
       it 'sets body' do
         expect {
-          subject.send(:created)
+          subject.resolve
         }.to change { subject.reload.body }.from(nil).to(payload['comment']['body'])
       end
     end
 
     describe '#edited' do
+      before { subject.payload['action'] = 'edited' }
+
       it 'edits body' do
         body = payload['comment']['body']
```

Comment:
@hdamico, `payload` has several nested fields named `body`.
Please rename the variable `body` to `comment_body` to disambiguate its intention.

Reply:
I didn't understand your point, do you mean the variable name?

Reply:
Yes, I meant the variable name, to make it explicit that is the body of the comment.

Reply:
Mmm.. In that context i don't think is it necessary to put `comment_body`. We know that we are in the `ReviewComment` model, and in the describe says that it changes the body.

Reply:
I was thinking in reading

`subject.update!(body: body)`

or

`}.to change { subject.reload.body }.from(body).to(payload['changes']['body'])`

in isolation.

While it is true that the context disambiguates the meaning of the variable, naming it

`subject.update!(body: comment_body)`

or

`}.to change { subject.reload.body }.from(comment_body).to(payload['changes']['body'])`

makes it explicit.

It was a suggestion anyway.

Reply:
Ok, i will!